tpp-app/__tests__/App.js

```diff
@@ -1,10 +0,0 @@
-import 'react-native';
```

Comment:
i removed this test because its causing issues with me mocking async-storage in CalculationServiceTests.js also it's not really doing anything